src/browser/connection/index.ts

```diff
@@ -59,6 +59,10 @@ interface ProviderMetaInfoOptions {
     appendToUserAgent?: boolean;
 }
 
+export interface BrowserCloseData {
```

Comment:
I proposing rename `BrowserCloseData` interface to `BrowserClosingInfo` for consistency (`BrowserInfo` and `ProviderMetaInfoOptions`).